google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/RpcViews.java

```diff
@@ -33,15 +33,32 @@
           RpcViewConstants.BIGTABLE_ATTEMPT_LATENCY_VIEW,
           RpcViewConstants.BIGTABLE_ATTEMPTS_PER_OP_VIEW);
 
+  private static final ImmutableSet<View> GFE_VIEW_SET =
+      ImmutableSet.of(
+          RpcViewConstants.BIGTABLE_GFE_LATENCY_VIEW,
+          RpcViewConstants.BIGTABLE_GFE_HEADER_MISSING_COUNT_VIEW);
+
   /** Registers all Bigtable specific views. */
   public static void registerBigtableClientViews() {
     registerBigtableClientViews(Stats.getViewManager());
   }
 
+  /** Register views for GFE metrics. */
```

Comment:
Please add a lot more docs explaining why you'd wanna do this. Also,`@BetaApi`

Reply:
RpcViews class is already marked with @BetaApi so I'm skipping adding it here